components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -132,95 +141,125 @@ export default function ProjectSettingsView() {
                     Update Name
                 </Button>
             </form>
-            <Heading2 className="mt-12">Prebuilds</Heading2>
-            <Subheading>Choose the workspace machine type for your prebuilds.</Subheading>
-            <div className="max-w-md">
-                <SelectWorkspaceClassComponent
-                    selectedWorkspaceClass={project.settings?.workspaceClasses?.prebuild}
-                    onSelectionChange={setWorkspaceClassForPrebuild}
+            <div className="">
+                <Heading2 className="mt-12">Prebuilds</Heading2>
+                <CheckboxInputField
+                    label="Enable prebuilds"
+                    hint="You need to have admin permissions on the repository in order to install a webhook for prebuilds."
```

Comment:
Thanks, @AlexTugarev !

**thought(blocking):** I assume that clicking enable will install the webhook? If so, that's not super obvious to me as a user. Mentioning the "webhook" at all here feels confusing, could we leave it up to the error? e.g. the user clicks to enable, if incorrect permissions show an error with an action? There feels like too much information that we'd need to contain here, e.g. we'd also need to mention that this overrides the `init` and `before` steps of your `.gitpod.yml`. @gtsiolis -> Any chance we could do a quick mockup and iterate on the copy text for @AlexTugarev? Happy to do a quick sync tomorrow morning if needed to get this unblocked.

Reply:
> I assume that clicking enable will install the webhook?

yes

> If so, that's not super obvious to me as a user.

that's not a change to current state at all. in fact, you would actually get feedback if it fails. today, if it fails, you wouldn't even notice as everything happens behind the scene and there is no feedback. 

> Mentioning the "webhook" at all here feels confusing, could we leave it up to the error? e.g. the user clicks to enable, if incorrect permissions show an error with an action?

yes. I like that! let's simplify the hint, not mentioning the webhook and improve the error message. 
CTA? which one? I think the most we can do is point out to check write permissions in Gitpod and admin permissions on the project at the SCM provider.

👍🏻 for the sync, but I don't see a blocker her TBH. this is where we need input, thus please fill in the gaps 🙏🏻